DESCRIPTION

```diff
@@ -1,8 +1,7 @@
 Package: x13binary
 Type: Package
 Title: Provide the 'x13ashtml' Seasonal Adjustment Binary
-Version: 1.1.57-4
-Date: 2023-09-07
+Version: 1.1.60
```

Comment:
Date is indeed optional. I find it helpful.  Your call.

Reply:
It saves me from forgetting to update